src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form"
+      action="/search"
+      method="get"
+      data-qa="big"
+    >
+      <img
+      class="search-icon"
+      src="images/Search.svg"
+      alt="Search-bar"
+      >
+
+      <label for="input">
+        <input type="text"
+          data-qa="keypress"
+          class="search-input"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
```

Comment:
```suggestion
        <input 
          type="text"
          data-qa="keypress"
          class="search-input"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```
Each attribute should be on a new line